app.py

```diff
@@ -229,6 +233,98 @@ def button_pushed():
         logger.warning("ATM shutdown (5 times button)")
         os.system("sudo shutdown -h now")
 
+    if config.PUSHES == 9:
```

Comment:
was already fully implemented in the display align test

Reply:
Unfortunately I saw that too late. Now I've brought all the ads into the call in app.py as well.